### ISSUE TITLE
Invalid or corrupt jarfile gatk-package-4.4.0.0-local.jar and incompatibility with GRaDLE

### ISSUE THREAD

**Issue by AthenACHY** [broadinstitute/gatk#8432]
## Bug Report

Hi there,
So I downloaded the gatk-4.4-0.0.zip and unzipped it for using gatk. I also created the conda env using the gatkcondaenv.yml and used conda to install java "1.7.0_91". But when I run ./gatk --list I got this error message: 

`
./gatk --list
Using GATK jar /home/athchu/bin/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar
Running:
    java -Dsamjdk.use_async_io_read_samtools=false -Dsamjdk.use_async_io_write_samtools=true -Dsamjdk.use_async_io_write_tribble=false -Dsamjdk.compression_level=2 -jar /home/athchu/bin/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar --help
Error: Invalid or corrupt jarfile /home/athchu/bin/gatk-4.4.0.0/gatk-package-4.4.0.0-local.jar
`
Next, I moved on to git clone the gatk repository, trying to build gatk. Again, I stay in the java "1.7.0_91" gatk env that I already created. But I got this error msg this time:
`
./gradlew localJar
Gradle 7.5.1 requires Java 1.8 or later to run. You are currently using Java 1.7.
`
When I switch back to the server default java (1.8.0_292-b10), i got another error msg.
`
java -version
openjdk version "1.8.0_292"
OpenJDK Runtime Environment (build 1.8.0_292-b10)
OpenJDK 64-Bit Server VM (build 25.292-b10, mixed mode)

./gradlew localJar

> Configure project :
Warning: using Java 1.8 but only Java 17 has been tested.

FAILURE: Build failed with an exception.

* Where:
Build file '/home/athchu/bin/gatk/build.gradle' line: 141

* What went wrong:
A problem occurred evaluating root project 'gatk'.
> A Java 17 compatible (Java 17 or later) version is required to build GATK, but 1.8 was found. See https://github.com/broadinstitute/gatk#building for information on how to build GATK.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org
`
So to sum up, my issues are :
1) downloaded gatk-4.4.0.0 but it contained invalid jar file and i cannot run GATK
2) following the github instruction, I cannot build gatk under java1.7.0_91, because it is incompatible to GRADLE7.5.1.
3) built gatk using java 1.8.0_292 failed because gatk is java 17 compatible.

Would you please advise on what I shall do? The docker installed in my sever doesn't not work....

**Comment by cmnbroad** [comment]
@AthenACHY It looks like (in all of these cases) you're trying to use `Java 1.7`, but GATK requires `Java 17`. Installing `Java 17` should solve the problem.
